### PR TITLE
Complete documentation with an eye on first release

### PR DESCRIPTION
**Pull request by pinin4fjords**
In this PR:

- Finalise documentation
- Remove relics of MultiQC (not currently being used)
- Updating citations
- Workflow tweak so that the report imports the citations rather than repeating them
- Last couple of minor TODOs

closes https://github.com/nf-core/differentialabundance/issues/22

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -6,58 +6,89 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+Differential analysis is a common task in a variety of use cases. In essence, all these use cases entail taking an input matrix containing features (e.g. genes) and observations (e.g. samples), comparing groups of observations in all or a subset of the features. The feature/ observation language here reflects our hope that this workflow will extend in future to encompass a variety of applications where an assumption of gene vs sample may not be a valid one- though that is the application to which the first release will apply.
```
Comment (WackerO) [inline docs/usage.md:9]:
```suggestion
Differential analysis is a common task in a variety of use cases. In essence, all these use cases entail taking an input matrix containing features (e.g. genes) and observations (e.g. samples), comparing groups of observations in all or a subset of the features. The feature/observation language here reflects our hope that this workflow will extend in future to encompass a variety of applications where an assumption of gene vs sample may not be a valid one - though that is the application to which the first release will apply.
```

File: docs/usage.md [hunk 2]
```diff
@@ -6,58 +6,89 @@
...
 
-## Samplesheet input
+With the above in mind, running this workflow requires:
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+- a matrix of quantifications with observations by column and features by row
+- a description of the observations such as a sample sheet from RNA-seq analysis
+- a description of the features, for our initial RNA-seq applicaiton this is simply the GTF file from which gene annotations can be derived
```
Comment (WackerO) [inline docs/usage.md:15]:
```suggestion
- a description of the features, for our initial RNA-seq application this is simply the GTF file from which gene annotations can be derived
```

**Review by WackerO — APPROVED** [review]
Minor corrections, other than that lgtm!

**Review by mashehu — CHANGES_REQUESTED** [review]

File: docs/output.md [hunk 3]
```diff
@@ -2,57 +2,49 @@
...
+This directory contains the main reporting output of the workflow.
 
 <details markdown="1">
 <summary>Output files</summary>
 
-- `fastqc/`
-  - `*_fastqc.html`: FastQC report containing quality metrics.
-  - `*_fastqc.zip`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+- `report/`
+  - `*.html`: an HTML report file named according to the value of `params.study_name`, containing graphical and tabular summary results for the workflow run.
```
[the remark this answers is not in the document]
Comment (pinin4fjords) [reply docs/output.md:15]: Thanks, but individual html plots have been removed, in favour of the single html report (which this line indicates).

[the remark this answers is not in the document]
Comment (mashehu) [reply docs/output.md:15]: okay, sorry, I couldn't find an explicit definition of the file name anywhere in your code. But reading the line again, it makes sense now.

**Review by mashehu — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks for the review @mashehu @WackerO

**Pull request merged**